src/test/java/gov/healthit/chpl/aqa/stepDefinitions/LoginSteps.java

```diff
@@ -54,10 +62,26 @@ public void logInAsAcb() {
 
     /**
      * Verify login attempt was successful.
+     * @param role as ACB or Admin
      */
-    @Then("^I should be logged in to CHPL as ACB Admin$")
-    public void verifyLoginWasSuccessful() {
+    @Then("^I should be logged in to CHPL as \"([^\"]*)\"$")
+    public void verifyLoginWasSuccessful(final String role) {
         String actualString = LoginPage.welcomeText(driver).getText();
         assertTrue(actualString.contains("Welcome"));
```

Comment:
This "Welcome" text should be extended to show something like "Welcome, <insert appropriate "name" variable here>"